### DISCUSSION TITLE
cs2 skin plugins

### DISCUSSION THREAD

**Discussion by hassann1122** [Nereziel/cs2-WeaponPaints#234] [General]
for some reason ive tried for like 5 hours straight to use the skin plugins for cs2 on dathost server and they still dont work and i cant get them to work if there is someone that could help me to get them work it would be amazing and i could even add a tip for them but i need help asap thank you in regards

**Comment by blowtilts** [comment]
Good afternoon friend, I managed to install the plug-in, what are your questions?

**Reply by PIKSELASI** [reply]
i put it in my server and commands dont work.How to make them work without website?

**Comment by blowtilts** [comment]
Hi, I haven't tested the commands but in fact, we always use the server
with the website. I have a gamer arena and people always use it through the
website. But today I'm going to try to use the commands and if I find an
error I can send you here.

Em seg., 24 de fev. de 2025, 22:02, PIKSELASI ***@***.***>
escreveu:

**Reply by PIKSELASI** [reply]
Okay,thanks on reply.See ehat you can do and send me message if you can.I would really like to fix commands so that i can play.Thanks for help we will stay in contact
